lib/textmeasure/textmeasure.go

```diff
@@ -138,7 +139,12 @@ func NewRuler() (*Ruler, error) {
 func (r *Ruler) addFontSize(font d2fonts.Font) {
 	sizeless := font
 	sizeless.Size = 0
-	face := truetype.NewFace(r.ttfs[sizeless], &truetype.Options{
+	ttf := r.ttfs[sizeless]
+	if ttf == nil {
```

Comment:
there's a second param in Go for these (https://stackoverflow.com/questions/2050391/how-to-check-if-a-map-contains-a-key-in-go)

Reply:
nice, thanks for the pointer! always annoys me when people don't get the return value in dotnet for `.TryGetValue` 😁